robot_ws/src/hello_world_robot/src/hello_world_robot/rotate.py

```diff
@@ -28,7 +28,9 @@ class Rotator(Node):
 
     def __init__(self):
         super().__init__('rotate')
-        self._cmd_pub = self.create_publisher(Twist, '/cmd_vel')
+        # For Foxy, qos_profile is a required field for create_publisher api
+        # For Dashing, qos_profile is optional and defaults to 10.
+        self._cmd_pub = self.create_publisher(Twist, '/cmd_vel', 10)
```

Comment:
https://github.com/ros2/rclpy/blob/dashing/rclpy/rclpy/node.py#L1027 seems to suggest dashing takes optional qos_profile parameter. To be safe, have you tested this does not break dashing?

Reply:
*I would suggest adding tests related description in the PR if possible